features/security_subsystem/FEATURE_SSS/api/trng_api.c

```diff
@@ -0,0 +1,40 @@
+/****************************************************************************
```

Comment:
General note: This PR adds a new subsystem called "security_subsystem" with a new feature called "SSS". This is something that's used for general features, not for board specific ones. 
Now, in this case, it seems that the feature added here is TRNG. We have a very well defined interface for that under hal/trng_api.h. Board specific implementations for such features should be placed under target code. For reference, you can look at the many boards implementing trng under "target" directory (code that includes "trng_api.h").

Reply:
Addressed in the latest fc422106e140aaddc7ef5e7869d45a966e3c78cc